worlds/sc2/Items.py

```diff
@@ -1322,22 +1322,54 @@ def get_full_item_list():
-    ItemNames.CORRUPTOR: ItemData(12 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 12, SC2Race.ZERG, classification=ItemClassification.progression, origin={"ext"}),
-    ItemNames.SCOURGE: ItemData(13 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 13, SC2Race.ZERG, classification=ItemClassification.progression, origin={"bw", "ext"}),
-    ItemNames.BROOD_QUEEN: ItemData(14 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 4, SC2Race.ZERG, classification=ItemClassification.progression, origin={"bw", "ext"}),
-    ItemNames.DEFILER: ItemData(15 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 14, SC2Race.ZERG, classification=ItemClassification.progression, origin={"bw"}),
+    ItemNames.ZERGLING: ItemData(0 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 0, SC2Race.ZERG, 
+                     classification=ItemClassification.progression, origin={"hots"}, 
+                     description="Fast inexpensive melee attacker. Can morph into a Baneling."),
+    ItemNames.SWARM_QUEEN: ItemData(1 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 1, SC2Race.ZERG, 
+                     classification=ItemClassification.progression, origin={"hots"}, 
+                     description="Ranged support caster. Can use the Spawn Creep Tumor and Rapid Transfusion abilities."),
+    ItemNames.ROACH: ItemData(2 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 2, SC2Race.ZERG, 
+                     classification=ItemClassification.progression, origin={"hots"}, 
+                     description="Durable short ranged attacker. Regenerates life quickly when burrowed."),
+    ItemNames.HYDRALISK: ItemData(3 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 3, SC2Race.ZERG, 
+                     classification=ItemClassification.progression, origin={"hots"}, 
+                     description="High-damage generalist ranged attacker."),
+    ItemNames.ZERGLING_BANELING_ASPECT: ItemData(4 + SC2HOTS_ITEM_ID_OFFSET, "Morph", 5, SC2Race.ZERG, 
+                     classification=ItemClassification.progression, origin={"hots"}, 
+                     description="Anti-ground suicide unit. Does damage over a small area on death."),
+    ItemNames.ABERRATION: ItemData(5 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 5, SC2Race.ZERG, 
+                     classification=ItemClassification.progression, origin={"hots"}, 
+                     description="Durable melee attacker that deals heavy damage."),
+    ItemNames.MUTALISK: ItemData(6 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 6, SC2Race.ZERG, 
+                     classification=ItemClassification.progression, origin={"hots"}, 
+                     description="Fragile flying attacker. Attacks bounce between targets."),
+    ItemNames.SWARM_HOST: ItemData(7 + SC2HOTS_ITEM_ID_OFFSET, "Unit", 7, SC2Race.ZERG, 
+                     classification=ItemClassification.progression, origin={"hots"}, 
+                     description="Siege unit that attacks while burrowed by continually spawning Locusts."),
```

Comment:
Technically the base Swarm Host roots, not burrows. Burrow is an upgrade for them. Maybe something like:
> Siege Unit that can continually spawn locusts when rooted in place.